### PR TITLE
Mention VS Code live preview

### PR DESCRIPTION
**Pull request by LPardue**
Coming from Microsoft, in my experience this extension seems subjectively better than other web server live preview extensions.

### CODE REVIEW

File: drafting-in-markdown.md [hunk 1]
```diff
@@ -47,3 +47,7 @@ Authors writing in Markdown may find the following tools helpful:
 
 ## kdwatch
 [kdwatch](https://github.com/cabo/kdwatch) is a locally installable live-update rendering display.
+
+## vscode-livepreview
+[vscode-livepreview](https://github.com/microsoft/vscode-livepreview) is a VS Code extension from Microsoft that can preview the HTML document rendering. It does this by hosting a local HTTP server that can be accessed via the internal VS Code browser or and external browser. Can be combined with any "build
```
Comment (ajeanmahoney) [inline drafting-in-markdown.md:52]: There's a small typo here: 
"or and external" / "or an external"

**Review by ajeanmahoney — CHANGES_REQUESTED** [review]

File: drafting-in-markdown.md (hunk 1 above)
Comment (LPardue) [inline drafting-in-markdown.md:52]:
```suggestion
[vscode-livepreview](https://github.com/microsoft/vscode-livepreview) is a VS Code extension from Microsoft that can preview the HTML document rendering. It does this by hosting a local HTTP server that can be accessed via the internal VS Code browser or an external browser. Can be combined with any "build
```

Comment (LPardue) [reply drafting-in-markdown.md]: nice catch, fixed thanks

**Comment by ajeanmahoney** [comment]
Thanks for making the update!

**Review by Darby101 — CHANGES_REQUESTED** [review]
- [ ]

**Pull request merged**